build.sbt

```diff
@@ -1176,8 +1176,15 @@ lazy val sbtClientProj = (project in file("client"))
     nativeImageReady := { () =>
       ()
     },
-    nativeImageVersion := "22.2.0",
-    nativeImageOutput := target.value / "bin" / "sbtn",
+    nativeImageVersion := "23.0",
+    nativeImageJvm := "graalvm-java23",
+    nativeImageOutput := {
+      val outputDir = (target.value / "bin").toPath
```

Comment:
Without explicitly creating the dir, the following Error is thrown

> Error: Writing image to non-existent directory /Users/runner/work/sbt/sbt/client/target/bin is not allowed. Create the missing directory if you want the image to be written to that location.

Example: https://github.com/Friendseeker/sbt/actions/runs/11008291362/job/30565658216